### PR TITLE
update envoy

### PR DESCRIPTION
**Pull request by yuval-k**

### CODE REVIEW

File: changelog/v1.4.3/update-envoy.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+changelog:
+- type: DEPENDENCY_BUMP
+  description: Update envoy to pull in a for transformation not working for AWS usptreams.
```
Comment (kdorosh) [inline changelog/v1.4.3/update-envoy.yaml:3]: pull in a fix?

File: changelog/v1.4.3/update-envoy.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+changelog:
+- type: DEPENDENCY_BUMP
+  description: Update envoy to pull in a fix for transformation not working for AWS usptreams.
```
Comment (srenatus) [inline changelog/v1.4.3/update-envoy.yaml:3]: typo: `s/usptreams/upstreams/`

**Pull request merged**